src/natcap/invest/utils.py

```diff
@@ -10,6 +10,7 @@
 import time
 
 import pandas
+import pint
```

Comment:
Exciting!

Reply:
🍺 !

Reply:
Had to unresolve this since it wouldn't let me add a new comment.

Do we still need to import `pint` here?

Reply:
whoops, removed `pint` in 542a7c39